docs/modules/ROOT/pages/exit.adoc

```diff
@@ -0,0 +1,21 @@
+= exit
+
+Exit command exits with the given code.
+
+Exit command can only be used in the interactive and scripting modes.
```

Comment:
for consistency:

NOTE: This command can only be used in interactive and batch mode.

Reply:
https://github.com/hazelcast/clc-docs/pull/5/commits/156434bc9a0369bc6690f0c7071ef6942ee01478